web/modules/slavnews_register/slavnews_register.module

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/**
+ * @file
+ * Primary module hooks for slavnews_register module.
+ */
+
+use Drupal\Core\Entity\EntityInterface;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Alter hook for form, adds additional field's.
+ */
+function slavnews_weather_form_user_register_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  if ($form_id == 'user_register_form') {
+    $form['#validate'][] = 'user_register_form_validate';
+  }
+
+}
+
+/**
+ * Function that validates Phone number.
+ */
+function user_register_form_validate(&$form, FormStateInterface $form_state) {
```

Comment:
You have to use the conventional names as wide as possible to avoid intersecting with already existing functions.
The best callback's name is (notice please underscore first) _YOURMODULENAME_WHATISDOING() e.g. in this case:
function _slavnews_weather_user_register_form_validate(&$form, &$form_state)